### DISCUSSION TITLE
has the org.apache.commons.text dependency been updated? re: CVE-2022-42889

### DISCUSSION THREAD

**Discussion by gtque** [reportportal/reportportal#1867] [Q&A]
A recent internal security scan we ran flagged our instance of reportportal as possibly vulnerable to CVE-2022-42889
https://nvd.nist.gov/vuln/detail/CVE-2022-42889

Seeing as the version of the apache commons text library with the fix was not published until September 24, is there a plan to release a new 5.7 release with the new version?

I suppose I should also ask if ReportPortal is using that library or is it transient dependency on something else that you would need to wait on?

Should I create an issue for this?

**Comment by HardNorth** [comment]
Thanks for pointing this out. But it is published 6 days ago, no, we don't have a fix right now.

**Reply by miracle8484** [reply]
Thx, It will be fixed in next release.

**Reply by gtque** [reply]
Thanks for the quick reply. I will pass that along to my company's security team.
